frontend/Views/Widgets/ViewGraphedStats/TestRunTable.svelte

```diff
@@ -39,14 +99,49 @@
             sort: (a: TestRun, b: TestRun) => a.version.localeCompare(b.version),
             width: "130px",
         },
+        {
+            key: "assignee",
+            label: "Assignee",
+            sort: (a: TestRun, b: TestRun) => (a.assignee || "").localeCompare(b.assignee || ""),
```

Comment:
Aren't those uuids or `"Unassigned"`? In that case this sort won't work at all if there are actually differently named assignees and will look random.